depends/packages/libmultiprocess.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=$(native_$(package)_sha256_hash)
 $(package)_dependencies=native_$(package) capnp
 
 define $(package)_config_cmds
-  $($(package)_cmake)
+  $($(package)_cmake) .
```

Comment:
In commit "build: specify cmake build dir for multiprocess depends build" (7c218dacd0e9602b8f755be42e96c49706f96305)

Note: I guess an alternate implementation could include the `.` in the `_cmake` variable, but this implementation seems good an more explicit. `_cmake` variable is defined here currently: https://github.com/bitcoin/bitcoin/blob/47bac475f044da0d8cd2e1f39fd2894f90dd0bd9/depends/funcs.mk#L172-L186